cfpb/consumerfinance.gov
# Add summary component

Ask article pages and career job posting pages used a heavily customized expandable that only showed up as a one-way expandable component at mobile sizes. This is vulnerable to breaking from changes in the underlying expandable component. This PR creates a dedicated component, "Summary Mobile," for handling this component's behavior and removes the dependency on expandables. 


## Additions

- Adds SummaryMobile organism.
- ~~Adds MaxHeightTransition for animating max-height CSS property.~~ moved to design system https://github.com/cfpb/design-system/pull/1161
- Adds test for Ask CFPB test to check that summary component hides and shows the component.


## Removals

- `read-more.js` for deprecated expandable-based component.


## Changes

- Adds parameter to FlyoutMenu to specify whether it should be expanded or collapsed at initialization-time.


## How to test this PR

1. Pull branch and run `./frontend.sh` and run local server.
2. Compare the following page types to production by resizing the pages down to mobile and clicking the read more button:
   - Ask article: http://localhost:8000/ask-cfpb/how-can-i-tell-if-a-friend-neighbor-or-family-member-is-a-victim-of-financial-exploitation-en-1933/
   - Career job posting: http://localhost:8000/about-us/careers/current-openings/management-and-program-analyst-21-cfpb-31-mpa/
** Button height is reduced by a few pixels, but the change is so slight I left it.**
3. Ensure the "Show full list of relevant parties" expandable is not impacted here http://localhost:8000/data-research/prepaid-accounts/search-agreements/detail/48439/
